engine/src/main/resources/templates/Rbac/insightsEmailBody.html

```diff
@@ -561,8 +561,8 @@
                                 <tr>
                                     <!--Red Hat logo-->
                                     <td align="center" class="rh-masthead__brand" bgcolor="#151515">
-                                        <a href="{environment.url('/')}" target="_blank" class="rh-masthead__brand-link">
-                                            <img src="{environment.url('/apps/frontend-assets/email-assets/logo_insights.jpg')}" alt="Red Hat logo" width="182" height="90" class="rh-masthead__brand-img" />
+                                        <a href="https://console.redhat.com/" target="_blank" class="rh-masthead__brand-link">
```

Comment:
I think this link is OK as is, IMO we only need to change the img src.

![Peek 2022-05-11 12-29](https://user-images.githubusercontent.com/3845764/167920710-adc59d04-c711-491a-b0e7-cd31bc3597c2.gif)

Reply:
Done 👍